assets/model_monitoring/components/src/model_performance_metrics/run.py

```diff
@@ -22,6 +22,16 @@ def run():
     parser.add_argument("--production_data", type=str, dest="predictions", required=True)
     parser.add_argument("--production_data_target_column", type=str,
                         dest="predictions_column_name", required=False, default=None)
+    parser.add_argument("--regression_rmse_threshold", type=str,
+                        dest="regression_rmse_threshold", required=False, default=None)
```

Comment:
If the default value is None, what's the behavior?

Reply:
If either regression or classification threshold is null, then it is the expected behavior: we are doing regression or classification model performance evaluation. If both of them are none(which is not expected), then the metrics is calculated, while our evaluation will skip the evaluation of these metrics and in the output, we will only have metrics value.